src/components/__tests__/FeedbackModal.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { fireEvent } from '@testing-library/react-native'
+import React from 'react'
+
+import labels from '../../constants/labels.json'
+import render from '../../testing/render'
+import FeedbackModal from '../FeedbackModal'
+
+describe('FeedbackModal', () => {
+  const onClose = jest.fn()
+
+  it('Button should be disabled for empty message', () => {
```

Comment:
test naming always starts with "should", please rename tests.

Reply:
should also be lowercase

Reply:
done